app/src/main/java/com/example/footprnt/HomeActivity.java

```diff
@@ -42,37 +48,90 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
         if (resultCode == Constants.RELOAD_USERPROFILE_FRAGMENT_REQUEST_CODE) {
             mFragment3.onActivityResult(requestCode, resultCode, data);
         }
+        if(resultCode == 302){
+            mFragment3.onActivityResult(requestCode, resultCode, data);
+        }
     }
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_home);
+        viewPager = (ViewPager) findViewById(R.id.viewpager);
+        shaddow = findViewById(R.id.shadow);
 
-        final BottomNavigationView navView = findViewById(R.id.nav_view);
-
+        navView = findViewById(R.id.nav_view);
+        final MediaPlayer mp = MediaPlayer.create(getApplicationContext(), R.raw.pop_two);
         navView.setOnNavigationItemSelectedListener(new BottomNavigationView.OnNavigationItemSelectedListener() {
             @Override
             public boolean onNavigationItemSelected(@NonNull MenuItem menuItem) {
+                mp.start();
                 Fragment fragment;
                 switch (menuItem.getItemId()) {
                     case R.id.navigation_home:
-                        fragment = mFragment1;
+                        viewPager.setCurrentItem(0);
                         break;
                     case R.id.navigation_dashboard:
-                        fragment = mFragment2;
+                        viewPager.setCurrentItem(1);
                         break;
                     case R.id.navigation_notifications:
+                        viewPager.setCurrentItem(2);
+                        break;
                     default:
-                        fragment = mFragment3;
+                        viewPager.setCurrentItem(0);
                         break;
+
                 }
-                mFragmentManager.beginTransaction().replace(R.id.flContainer, fragment).commit();
                 return true;
             }
         });
         navView.setSelectedItemId(R.id.navigation_home);
+
+
+        viewPager.addOnPageChangeListener(new ViewPager.OnPageChangeListener() {
+            @Override
+            public void onPageScrolled(int position, float positionOffset, int positionOffsetPixels) {
+
```

Comment:
You should never leave a method empty. Add a "// TODO" to indicate what work will be done or add a "// Empty method" to indicate that it's empty on purpose